src/test/java/org/openrewrite/java/testing/jmockit/JMockitToMockitoTest.java

```diff
@@ -33,32 +34,44 @@ public void defaults(RecipeSpec spec) {
             .classpathFromResources(new InMemoryExecutionContext(),
               "junit-jupiter-api-5.9",
               "jmockit-1.49",
-              "mockito-core-3.12"
+              "mockito-core-3.12",
+              "junit-4.13",
+              "mockito-junit-jupiter-3.12"
             ))
           .recipeFromResource(
             "/META-INF/rewrite/jmockit.yml",
-            "org.openrewrite.java.testing.mockito.JMockitToMockito"
-          );
+            "org.openrewrite.java.testing.jmockit.JMockitToMockito"
+          )
+          .typeValidationOptions(TypeValidation.none());
     }
 
     @Test
     void rewrite() {
         //language=java
         rewriteRun(
           java(
-            """                  
+            """
+              class MyObject {
+                  public String getSomeField() {
+                      return "X";
+                  }
+              }
+              """
+          ),
```

Comment:
What was the motivation behind adding this? I see no difference in the result with or without it.

Reply:
wanted to make sure the source code passes compile.